data/tests.yml

```diff
@@ -4,8 +4,8 @@
 ### - simple to complex
 ### - rural to urban
 ### - By infrastructure:
-###   - car-centric 
```

Comment:
We've hit "large diff" territory -- I'm trusting changes here blindly!

Reply:
yup, I highlighted with my comment the last meaningful change